mesheryctl/internal/cli/root/system/provider.go

```diff
@@ -380,21 +380,20 @@ mesheryctl system provider switch [provider]
 // To reset provider to default
 mesheryctl system provider reset
 	`,
-	RunE: func(cmd *cobra.Command, args []string) error {
+	Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
Thanks, @singh1203. 

Why not RunE?

Reply:
I moved args condition to Args field, RunE is there, Github diff is not able to show here.

Reply:
Weird... I still don't see `RunE`. Where did it `RunOff` to?